src/expr.ts

```diff
@@ -167,7 +167,7 @@ const fnMapDatasetToValue: FnMapDatasetToValue = {
         let streak = 0;
         let maxStreak = 0;
         for (let dataPoint of dataset) {
-            if (dataPoint.value !== null) {
+            if (dataPoint.value) {
```

Comment:
And this is why I "love" typescript (sarcasm). dataPoint.value is supposed to be a number field, however in reality Obsidian is passing in booleans as well.